cypress/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM cypress/included:5.2.0
+
+WORKDIR /root
+
+RUN apt-get install curl
```

Comment:
looks like this can be safely removed.

Reply:
the docker image already has this in - that's useful to know 👍

Reply:
Indeed it is. Good spot :)

Reply:
yep - circleci logs mentioned no update needed. happy to approve now :)